fastapi_sqla/_pytest_plugin.py

```diff
@@ -153,7 +153,9 @@ def async_sqlalchemy_url(db_url):
 
     @fixture
     def async_engine(async_sqlalchemy_url):
-        return create_async_engine(async_sqlalchemy_url)
+        engine = create_async_engine(async_sqlalchemy_url)
+        yield engine
+        engine.dispose()
```

Comment:
Should this be awaited? IIUC it's an async function

Reply:
indeed, it is. https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.AsyncEngine.dispose